README.md

```diff
@@ -106,6 +106,10 @@ Supported Rules
 
 A sample configuration file with all options is available [here](https://github.com/palantir/tslint/blob/master/docs/sample.tslint.json).
 
+* `align` enforces vertical alignment. Rule options:
```

Comment:
Thanks for updating the README!
Please update the sample.tslint.json (in docs) with this new rule and its options as well.